gr-fec/python/fec/LDPC/Generate_LDPC_matrix_functions.py

```diff
@@ -27,12 +27,12 @@ def read_alist_file(filename):
 
   myfile    = open(filename,'r')
   data      = myfile.readlines()
-  size      = string.split(data[0])
+  size      = data[0].split()
   numCols   = int(size[0])
   numRows   = int(size[1])
```

Comment:
I'd say this change is already an improvement. While you're at it, i suggest to restructure this as:
```python
with open(filename, 'r') as myfile:
    numCols, numRows = parse_alist_header(data[0])
```
This should avoid errors in case smth went wrong with the filename. Also, it makes the code more verbose.

Same goes for the write alist file function that tries to do error handling.

Reply:
I have added a commit with this restructuring

Reply:
Thanks!